erigon-lib/state/aggregator.go

```diff
@@ -1413,7 +1407,7 @@ func (ac *AggregatorRoTx) mergeFiles(ctx context.Context, files *SelectedStaticF
 	return mf, err
 }
 
-func (a *Aggregator) integrateMergedDirtyFiles(outs *SelectedStaticFilesV3, in *MergedFilesV3) {
+func (a *Aggregator) IntegrateMergedDirtyFiles(outs *SelectedStaticFilesV3, in *MergedFilesV3) {
```

Comment:
why do we need this and `IntegrateDirtyFiles` as exported? for later move?

Reply:
No real reason. 
Just i follow “take locks in public methods” convention to make private methods composable (take lock - run many private methods - release lock)

can be private